Update Document sandbox samples with new runtime and SDK naming (#5)
AdobeDocs/express-add-on-samples: document-sandbox-samples/communication-iframe-documentSandbox/README.md

## About

> This Add-on demonstrates how to get started using the document sandbox based add-ons and communicate between the iframe (index.html) running your add-on and the document sandbox (code.js). 

Reviewer: ```suggestion
This Add-on demonstrates how to communicate between code in the UI iframe (index.html) and the document model sandbox (code.js).
```
